src/flask_api.py

```diff
@@ -0,0 +1,187 @@
+from flask import Flask, request
+import json
+import numpy as np
+import gdown
+
+from fastai.vision import load_learner
+
+from . import predict as pred
+from . import transform_data as td
+
+# import predict as pred
+# import transform_data as td
+
+import base64
+import io
+from PIL import Image
+from pathlib import Path
+
+app = Flask(__name__)
+host = 'localhost'
+port = '5000'
+
+# load model
+try:
+    model_path = Path(__file__).parents[1]
+    learn = load_learner(model_path, 'stage-2_bs16.pkl')
+except IOError:
```

Comment:
A better way to do this would be to explicitly check if the file exists (you can treat files as path object as well), and download if not. Less hacky than relying on an IOError.